### ISSUE TITLE
Improve data localization

### ISSUE THREAD

**Issue by dirk-qualisys** [esdscom/sdscom-xml#201]
Some enum types are not only used to identify items, but are in practice printed directly in the SDS. This fails whenever national translations of values are necessary, for example LD50 becomes DL50 in French. SDScom often does not have fields to hold the printable data, which requires additional processing/translating by the receiving system. Add such fields where necessary.

**Comment by dirk-qualisys** [comment]
Some thoughts... 

Solution depends on the implementation of issue #210 . If multiple languages are supported, the result must be 
a) a phrase, probably forcing IT systems to create phrases for such info, or 
b) a multi-language text type for non-phrase content, because IT systems might want to maintain their own logic how to map enumerations to printable content. On the other hand, if they do so, they could simply not import the phrase. And with a multi-language free text type, we increase the complexity of SDScom and loose the _option_ of handling this as a phrase.

So I prefer b). Which makes me think that even for just one language in a file, it would be good to have a phrase in order to make the printable representation properly available in all languages. So I prefer b) in every case.

**Comment by dirk-qualisys** [comment]
As version 5.0.0 will stick to just one language but cover multiple areas in one `<Datasheet>` tag, the only way to cover multiple areas are phrases.

Enumerations with just numbers or otherwise alphanumerical patterns that do not form an acronym do not need translations, e.g. AgpaPictogramEnum, RmmPictogramEnum.

Administrative information that does not need direct printing in the SDS (but may influence the print template or processing) does not need translations either, e.g. RoleDescriptionEnum, CompositionTypeEnum, SymbolEnum.

Also, enumerations that cover national regulations will not need translations (as long as the country has no more than 1 language or  the enum value is understood in all languages, e.g. ExposureLimitValueTypeEnum.

The following enumerations were identified for translations: DoseEnum, EffectLevelGroupEnum, ExposureRouteEnum, EffectLevelTypeEnum, EffectLevelTypeEnum, ViscosityTypeEnum, SolubilityMediumEnum, StateUnderStandardConditionsEnum, ExposureLimitFractionEnum, AssessmentEnum, InvestigationMethodEnum, ReproductiveToxicityTypeEnum.

Also, category systems and Iuclid compatible enumerations should have optional translations: SectorOfUseCodeEnum, ProductCategoryCodeEnum, ProcessCategoryCodeEnum, EnvironmentalReleaseCategoryCodeEnum, SpecificEnvironmentalReleaseCategoryCodeEnum, ArticleCategoryCodeEnumEuPCSEnum, FormEnum, ColourEnum, ColourIntensityEnum, PackagingTypeEnum, PackagingMaterialEnum, SubstanceTypeEnum, FormulationTypeEnum, ComponentFunctionEnum.

**Comment by dirk-qualisys** [comment]
Around half of the fields had according translation fields anyway, but not in a consistent manner. I have now applied the following rules:

- If any changes were necessary at all, the enum field and the phrase field form a node together. In this node, the enum field is mandatory - the node itself is usually optional so that the value can be skipped, but this defines the hierarchy so that recipients can rely on the enum as before.
- If that node is necessary more than once, I defined a type `Xxx` for the enum `XxxEnum`.
- If the enum contains "other" or is meant as an incomplete list, the phrase field is called `XxxDescription`. If it is meant as a 1:1 transcription of the enum, it is called `XxxFulltext`. In the latter case, the phrase field may be restricted to just 1 phrase, depending on the enum content.
